src/store/activity/actions.js

```diff
@@ -22,7 +22,7 @@ import { formatCirclesValue } from '~/utils/format';
 
 const { ActivityTypes, ActivityFilterTypes } = core.activity;
 
-export const PAGE_SIZE = 2;
+export const PAGE_SIZE = 10;
```

Comment:
Can you please make sure that this is the same parameter as is sent to the core request, so that if changed, the pagination to the request is also changed?
Otherwise if we change this later on, some notifications will not be shown or similar, if I'm not mistaken. I haven't thought it through completely, but it can mess up behaviour if the next person editing thinks this changes page size in the request. It is important to use propagate such a variable to the request, otherwise it is deceiving

Reply:
as I wrote before it is - it is in line 311. Code below:

```const { activities, lastTimestamp } = await core.activity.getLatest(
        safe.currentAccount,
        ActivityFilterTypes.DISABLED,
        PAGE_SIZE,
        activity.lastTimestamp,
        offset,
        otherSafeAddress,
      );```